provider/datagen/src/transform/cldr/cldr_serde/mod.rs

```diff
@@ -23,4 +23,5 @@ pub mod plurals;
 pub mod region_displaynames;
 pub mod script_displaynames;
 pub mod time_zones;
+pub mod variant_displaynames;
```

Comment:
nit: `displaynames/{variant,region,script,language}.rs` would be a better structure.

Reply:
I agree. However, this would require changing some code in datagen/src/transform/cldr as well. I will follow up with a PR after submitting this.